lib/wasmex/native.ex

```diff
@@ -5,23 +5,9 @@ defmodule Wasmex.Native do
   version = mix_config[:version]
   github_url = mix_config[:package][:links]["GitHub"]
 
-  use RustlerPrecompiled,
+  use Rustler,
     otp_app: :wasmex,
-    base_url: "#{github_url}/releases/download/v#{version}",
-    version: version,
-    targets: ~w(
-      aarch64-apple-darwin
-      aarch64-unknown-linux-gnu
-      aarch64-unknown-linux-musl
-      riscv64gc-unknown-linux-gnu
-      x86_64-apple-darwin
-      x86_64-pc-windows-gnu
-      x86_64-pc-windows-msvc
-      x86_64-unknown-freebsd
-      x86_64-unknown-linux-gnu
-      x86_64-unknown-linux-musl
-    ),
-    force_build: System.get_env("WASMEX_BUILD") in ["1", "true"]
+    mode: :debug
```

Comment:
I imagine you do this for local development, right? in the finished PR I'd remove this change (if this is a proposal, let's maybe move it to a separate PR)

Reply:
Yep, just leftover just from development :) Will remove